php/commands/media.php

```diff
@@ -203,15 +207,15 @@ private function _make_copy( $path ) {
 		return $filename;
 	}
 
-	private function _process_regeneration( $id, $skip_delete = false ) {
+	private function _process_regeneration( $id, $skip_delete = false, $only_missing = false ) {
 		$image = get_post( $id );
 
 		$fullsizepath = get_attached_file( $image->ID );
 
-		$att_desc = sprintf( '"%1$s" (ID %2$d).', get_the_title( $image->ID ), $image->ID );
+		$att_desc = sprintf( '"%1$s" (ID %2$d)', get_the_title( $image->ID ), $image->ID );
 
 		if ( false === $fullsizepath || !file_exists( $fullsizepath ) ) {
-			WP_CLI::warning( "Can't find $att_desc" );
+			WP_CLI::warning( "Can't find $att_desc." );
```

Comment:
Hi, I think that the variable shouldn't have the trailing full stop in it as it can't be used in the middle of sentence, that's why I changed it. Is it a big issue?